### PR TITLE
Update readme.md and further links in rm project material

### LINKED ISSUE #40: Update links in README.md file
**Problem description**
Missing and outdate links / information in the README.md file need to be fixed.

(edit/added) The link within API-Readiness-Checklist.md to the wiki is also pointing not to the correct page.

**Expected action**
Update README.md file and API-Readiness-Checklist.md

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by tanjadegroot**
#### What type of PR is this?

Add one of the following kinds:
* cleanup
* documentation

#### What this PR does / why we need it:
* Corrects links in README.md and in other RM documentation files.
* Moves CHANGELOG_TEMPLATE.md to SupportingDocuments folder

#### Which issue(s) this PR fixes:

Fixes #40 

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,32 +13,36 @@ Repository to describe, develop, document and test the Release Management proces
 * Guidelines and assets for Release Management in CAMARA
 * Describe, develop, document and test the deliverables
 * Started: November 2023
+* First release: r1.0.0 (July 2024)
 
-## Release Information
+## Release Process Information
 
-This projects constains the following (documentation folder):
+This projects constains the following documentation:
```
Comment (hdamker) [inline README.md:20]:
```suggestion
This projects contains the following documentation:
```

File: README.md [hunk 2]
```diff
@@ -13,32 +13,36 @@ Repository to describe, develop, document and test the Release Management proces
...
 
-* API release guidelines
-* API readiness checklist 
+* [API Release Guidelines](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md)
```
Comment (hdamker) [inline README.md:22]: I propose to use relative links ... they would still work if we release/tag the repository without a change.

```suggestion
* [API Release Guidelines](/documentation/API_Release_Guidelines.md)
```

File: README.md [hunk 3]
```diff
@@ -13,32 +13,36 @@ Repository to describe, develop, document and test the Release Management proces
...
+* [API Readiness Checklist (template for copying)](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md)
```
Comment (hdamker) [inline README.md:23]:
```suggestion
* [API Readiness Checklist (template for copying)](/documentation/API-Readiness-Checklist.md)
```

File: README.md [hunk 4]
```diff
@@ -13,32 +13,36 @@ Repository to describe, develop, document and test the Release Management proces
...
+* [CHANGELOG (template for copying)](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md)
 
-More detailed information can be found in the below wiki pages.
+An example of the CHANGELOG created per the release management process is here: [CHANGELOG EXAMPLE](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/SupportingDocuments/CHANGELOG_EXAMPLE.MD)
```
Comment (hdamker) [inline README.md:26]:
```suggestion
An example of the CHANGELOG created per the release management process is here: [CHANGELOG EXAMPLE](/documentation/SupportingDocuments/CHANGELOG_EXAMPLE.md)
```

File: documentation/API_Release_Guidelines.md [hunk 5]
```diff
@@ -108,24 +108,24 @@ This section lists the steps to release an API version. More details can be foun
...
 * update of the version information in the API OAS definition files (no "wip" in the version field and base URL of any of the API files).
-* update the `API-Readiness-Checklist.md` file ensuring all required release assets are available. If this file not yet available, create a local copy under the /documentation folder and prefix it with the respective API name (copy it from [ReleaseManagement/documentation/API-Readiness-Checklist.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md).
-* update the `CHANGELOG.md` file in the home of the API repository. If not yet available, copy it from [ReleaseManagement/documentation/CHANGELOG_TEMPLATE.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md). See also the example available there.
+* complete the `APIname-API-Readiness-Checklist.md` file ensuring all required release assets are available. If not yet available, copy the template [API-Readiness-Checklist.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md), and prefix it with the API name.
```
Comment (hdamker) [inline documentation/API_Release_Guidelines.md:119]:
```suggestion
* complete the `APIname-API-Readiness-Checklist.md` file ensuring all required release assets are available. If not yet available, copy the template [API-Readiness-Checklist.md](/documentation/API-Readiness-Checklist.md), and prefix it with the API name.
```

File: documentation/API_Release_Guidelines.md [hunk 6]
```diff
@@ -108,24 +108,24 @@ This section lists the steps to release an API version. More details can be foun
...
+* update the `CHANGELOG.md` file in the home of the API repository. If not yet available, copy the [CHANGELOG_TEMPLATE.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md). See also the example available in the [SupportingDocuments folder](https://github.com/camaraproject/ReleaseManagement/tree/main/documentation/SupportingDocuments).
```
Comment (hdamker) [inline documentation/API_Release_Guidelines.md:120]:
```suggestion
* update the `CHANGELOG.md` file in the home of the API repository. If not yet available, copy the [CHANGELOG_TEMPLATE.md](/documentation/CHANGELOG_TEMPLATE.md). See also the example available in the [SupportingDocuments folder](/documentation/SupportingDocuments).
```

**Review by hdamker — COMMENTED** [review]
My comments are all on the links: my recommendation would be to use relative links if pointing to the same repository.

Regarding the release number: really 1.0.0 aside of Commonalities with 0.4.0 and ICM with 0.2.0? Let's discuss within the RM call.

**Comment by tanjadegroot** [comment]
Thanks Herbert ! local links are much better indeed.
Let's close and merge this on the RM call tomorrow.

**Review by hdamker — APPROVED** [review]

**Pull request merged**